pkgs/applications/science/biology/bftools/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, lib, makeWrapper, fetchurl, unzip, openjdk }:
+
+stdenv.mkDerivation rec {
+  name = "bftools-${version}";
+  version = "5.9.2";
+
+  src = fetchurl {
+    url = "http://downloads.openmicroscopy.org/bio-formats/${version}/artifacts/bftools.zip";
+    sha256 = "02x8p07z7js41kgiblkr9k8v9v4ka8rm7pg3f1vy9s5p6hmpfvnx";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp $(find ./* -maxdepth 1 ! -path "*.bat") $out/
+  '';
+
+  postFixup = ''
+    for script in "$out"/bin/*; do
+      wrapProgram "$script" --prefix PATH : "${lib.makeBinPath [ openjdk ]}"
+    done
+  '';
+
+  buildInputs = [ unzip ];
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  meta = with stdenv.lib; {
+    description = "A bundle of scripts for using Bio-Formats on the command line with bioformats_package.jar already included";
+    license = licenses.gpl3;
```

```suggestion
    license = licenses.gpl2;
```
https://github.com/openmicroscopy/bioformats/blob/1b39cedbc7744731f1c40cebfef3182ac88c11d2/LICENSE.txt
http://downloads.openmicroscopy.org/bio-formats/5.9.2/artifacts/bioformats-5.9.2.tar.xz